src/web/index.html.ejs

```diff
@@ -19,6 +19,7 @@
     <link rel="icon" type="image/svg+xml" href="images/icon.svg" />
   </head>
   <body>
+    <div id="modal"></div>
```

Comment:
Not super happy adding another root-level element, but this seemed the easiest (intended?) usage of React portals